src/test/scala/com/codacy/analysis/cli/CodacyClientSpec.scala

```diff
@@ -218,7 +218,7 @@ class CodacyClientSpec extends Specification with NoLanguageFeatures with Mockit
     val mockedHttpHelper = mock[HttpHelper]
     val response: Either[ParsingFailure, Json] =
       if (success)
-        parse("""{ "ignoredPaths": [], "projectExtensions": [], "toolConfiguration": [] }""")
+        parse("""{ "ignoredPaths": [], "defaultRegexIgnores": [], "projectExtensions": [], "toolConfiguration": [] }""")
```

Comment:
if you decide to change the "defaultRegexIgnores" name on the api, don't forget to also change it here.